pyke/targetpixelfile.py

```diff
@@ -47,14 +47,18 @@ class KeplerTargetPixelFile(TargetPixelFile):
         Kepler pipeline's aperture if the value 'kepler-pipeline' is passed.
         The default behaviour is to use all pixels.
     quality_bitmask : int
-        Bitmask specifying quality flags of cadences that should be ignored.
+        Bitmask specifying quality flags of cadences that should be ignored:
+            default: recommended quality mask
+            conservative: removes more flags, known to remove good data
+            hard: removes all data that has been flagged
```

Comment:
I have no strong feelings about this, but I quite like the convention we have right now, i.e., capital letters for these constants.

Reply:
Sure thing I don't have strong feelings either

Reply:
Also, the name postfix "_BITMASK" helps to clarify the meaning on this

Reply:
I think this is clear how it is, the call is 

```quality_mask = 'default'```

which I think makes it clear the mask applies to quality flags. You can still specify in the old way using

```quality_bitmask = KeplerQualityFlag.DEFAULT_BITMASK```